File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.java
class Solution {
    public int getMinDistance(int[] arr, int target, int start) {
        int min=Integer.MAX_VALUE;
        for(int i=0;i<arr.length;i++){
            if(arr[i]==target){
                min=Math.min(min,Math.abs(i-start));
            }
        }
        return min;
    }
}

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.c
int getMinDistance(int* arr, int numsSize, int target, int start) {
    int min=INT_MAX;
    for(int i=0;i<numsSize;i++){
        if(arr[i]==target){
            int d=abs(i-start);
            if(d<min){
                min=d;
            }
        }
    }
    return min;
}